tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -649,7 +650,7 @@ static __always_inline u32 get_task_ns_pid(struct task_struct *task)
     // kernel 4.14-4.18
     return READ_KERN(READ_KERN(task->pids[PIDTYPE_PID].pid)->numbers[level].nr);
 #else
-    // kernel 4.19 onwards
+    // kernel 4.19
```

Comment:
not only 4.19

Reply:
https://github.com/aquasecurity/tracee/pull/1185